### PR TITLE
Maker Wallet SOL and DC

### PR DESCRIPTION
**Pull request by samgutentag**

### CODE REVIEW

File: docs/solana/migration/hotspot-maker.mdx [hunk 1]
```diff
@@ -317,10 +317,69 @@ like to use the library, the setup guide can be found here
...
-<!-- TODO: We need to create an easy way for makers to get sol and data credits into their accounts -->
-
-```
-
-```
+## Data Credits and SOL for Onboarding
+
+:::info
+
+Maker Wallets will need both `SOL` and `DC` for transactions post Migration.
```
Comment (jthiller) [inline docs/solana/migration/hotspot-maker.mdx]: Lets clarify that SOL is for the Solana transaction and the DC is for the onboarding fees.

File: docs/solana/migration/hotspot-maker.mdx [hunk 2]
```diff
@@ -317,10 +317,69 @@ like to use the library, the setup guide can be found here
...
+
+:::
+
+### Maker Wallet SOL
+
+Maker Wallets will need `SOL` to pay transaction fees and for minting new merkle trees if creating
+double the current amount of Hotspots in their existing fleet.
```
Comment (jthiller) [inline docs/solana/migration/hotspot-maker.mdx]: This is kinda confusing. What is meant by 'creating double the current amount of Hotspots'?

This is also the first time we mention a merkle tree which is going to be a foreign concept to most of these radio manufacturers.

Comment (jthiller) [reply docs/solana/migration/hotspot-maker.mdx]: Maybe we just frame this as Hotspot manufacturers being responsible for 'minting' the Hotspot NFT? Should specify the rough per-Hotspot cost for this action.

File: docs/solana/migration/hotspot-maker.mdx [hunk 3]
```diff
@@ -317,10 +317,69 @@ like to use the library, the setup guide can be found here
...
+
+- For most makers, ~10 SOL is recommended.
+- Makers with large fleets (300k+ Hotspots) ~60 SOL is recommended.
+- For more insight to the amount of SOL needed, check out
+  [https://compressed.app/](https://compressed.app/)
+  - Enter the number of Hotspots to mint in the `How many compressed NFTs do you want to store?`
+    field.
+  - The `Uncompressed equivalent` value is a good approximation, round up when in doubt.
```
Comment (jthiller) [inline docs/solana/migration/hotspot-maker.mdx]: We might want to leave this calculator out. It's confusing that we are constantly talking about compression NFTs, but then we tell them to ignore that pricing here.

File: docs/solana/migration/hotspot-maker.mdx [hunk 4]
```diff
@@ -317,10 +317,69 @@ like to use the library, the setup guide can be found here
...
+
+<div className="img-center">
+  <img
+    alt="Helium Wallet app token swap screen"
+    className="appscreen"
+    src={useBaseUrl('img/solana/compressed-nft-calculator.png')}
+  />
+</div>
+
+> Minting 10,000 Hotspots would require ~1.134 SOL in this example.
```
Comment (jthiller) [inline docs/solana/migration/hotspot-maker.mdx]: can probably replace the calculator with a simple table
1,000 = X
10,000 = X
100,000 = X

File: docs/solana/migration/hotspot-maker.mdx [hunk 5]
```diff
@@ -317,10 +317,69 @@ like to use the library, the setup guide can be found here
...
+
+### Maker Wallet DC
+
+Maker Wallets will need `DC` to pay onboarding and assert fees. DC can be aquired by either
```
Comment (jthiller) [inline docs/solana/migration/hotspot-maker.mdx]:
```suggestion
Maker Wallets will need `DC` to pay Helium onboarding and assert fees. DC can be aquired by either
```

**Review by jthiller — CHANGES_REQUESTED** [review]
A bit of clarity on the comments then I think we're good to go.

File: docs/solana/migration/hotspot-maker.mdx (hunk 2 above)
Comment (samgutentag) [reply docs/solana/migration/hotspot-maker.mdx]: think i trimmed it down well, let me know how this looks

**Review by jthiller — APPROVED** [review]
ty for the changes!

**Pull request merged**